src/core/Akka.Remote.TestKit/Proto/ProtobufEncoder.cs

```diff
@@ -15,7 +15,7 @@ namespace Akka.Remote.TestKit.Proto
     /// <summary>
     /// Encodes a generic object into a <see cref="IByteBuf"/> using Google protobufs
     /// </summary>
-    public class ProtobufEncoder 
+    /*TODO: this class is not used*/public class ProtobufEncoder 
```

Comment:
Not used.  @Aaronontheweb you are the first in the git history, what is this for?